scenarios/aws/eks/fakeintake.go

```diff
@@ -0,0 +1,16 @@
+package eks
+
+import (
+	ddfakeintake "github.com/DataDog/test-infra-definitions/components/datadog/fakeintake"
+	"github.com/DataDog/test-infra-definitions/resources/aws"
+	"github.com/DataDog/test-infra-definitions/resources/aws/ecs"
+)
+
+func NewEcsFakeintake(env aws.Environment) (*ddfakeintake.ConnectionExporter, error) {
+	ipAddress, err := ecs.FargateServiceFakeintake(env)
```

Comment:
Oh nice, in the end you could reuse the same deployment used for VMs, editing helm values to configure the agent to talk to fakeintake. Cool !

Reply:
There’s undoubtedly pros and cons for both solutions: putting the fakeintake in ECS as it is already implemented today and used for the VM scenario, and putting the fakeintake in Kubernetes EKS.

I thought I would start with the simpler solution, that favors code re-use and generates the fewer maintenance.
We can always reconsider this choice later and move the fakeintake directly inside EKS if it happens to bring value.
But today, I’m happy with reusing the existing fakeintake deployment in ECS Fargate.